<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsGroupBox</class>
 <widget class="QGroupBox" name="SettingsGroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>rFactor2 Settings</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QCheckBox" name="checkBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Use custom settings for this car</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Feedback Tuning</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>TorqueMapping:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QSpinBox" name="max_steeringArmForce">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This setting is like the FFB ingame slider in iR if you use linear mode and Nm. To achieve &lt;span style=&quot; font-weight:600;&quot;&gt;1:1 output set this to your motor's torque&lt;/span&gt;.&lt;/p&gt;&lt;p&gt;If you want weaker effects then set this value to something like 30Nm or more, if your motor is only capable of 5Nm than set this to 5Nm.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="suffix">
         <string> Nm</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Smoothing:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="smoothing_spinBox">
        <property name="suffix">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="smoothing_slider">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Basically the same as the setting in app.ini&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;100% = maximum Smoothing (running average over the past 200ms)&lt;/p&gt;&lt;p&gt;50% = medium Smoothing (50% of current FFB mixed in the average of the past 200ms)&lt;/p&gt;&lt;p&gt;0% = no Smoothing&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Helpful especially for cars that have a lot of jitter coming from the tire model. It is recommended to use the lowest value that doesn't produce an uncomfortable amount of jitter for the most direct experience.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>* Set this to your motor's torque for 1:1 output.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkBox_invert">
     <property name="text">
      <string>Invert FFB</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
